test-files/cmd/slow/ConvertCmdTest.sh.stdout

```diff
@@ -1,8 +1,8 @@
 PBF to OSM...
```

Comment:
The only thing actually changed with this test output is that you no longer see warnings for invalid xml chars, since they are being encoded now instead.